Makefile

```diff
@@ -8,6 +8,9 @@ init:
 	curl -L https://github.com/uber/prototool/releases/download/v1.10.0/prototool-$(shell uname -s)-$(shell uname -m) -o ./bin/prototool
 	chmod +x ./bin/prototool
 
+	curl -L  https://github.com/go-swagger/go-swagger/releases/download/v0.25.0/swagger_$(shell uname | tr '[:upper:]' '[:lower:]')_amd64 -o ./bin/swagger
```

Comment:
It might be easier to install with Go modules

Reply:
As long as installation it's already there I vote to leave it